### PR TITLE
[Feature] add zh localization font

### PR DESCRIPTION
**Pull request by brain-frog**
## What are the changes?

Add a new font fallback for chinese localization

## Why am I doing these changes?

Chinese font size has been cut off and looking bad for TOO LONG

## What did change?

Added a font, updated i18n detection options, made some constants for the text (refactor coming soon I hope) 

### Screenshots/Videos

new-font

## How to test the changes?

set your language to one of the chinese ones and look around

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Review by Tempo-anon — APPROVED** [review]

File: src/plugins/i18n.ts [hunk 1]
```diff
@@ -94,44 +94,22 @@ const alternativeFonts = {
...
+function initFonts() {
+  Object.keys(alternativeFonts).forEach((language: string) => {
+    alternativeFonts[language].forEach((fontface: FontFace) => {
+      fontface.load().then(f => document.fonts.add(f)).catch(e => console.error(e));
     });
   });
 }
 
-export function initI18n(): void {
+export async function initI18n(): Promise<void> {
```
Comment (Greenlamp2) [inline src/plugins/i18n.ts:105]: i think you never return a Promise

Comment (brain-frog) [reply src/plugins/i18n.ts:105]: you have to set it to Promise<void> if its async
the point is to await the i18n init

Comment (Greenlamp2) [reply src/plugins/i18n.ts:105]: you never await init18n anywhere though.
You don't have to set it to Promise if its async, you have to set it if you want to return an awaitable

Comment (Greenlamp2) [reply src/plugins/i18n.ts:105]: "You don't have to set it to Promise if its async"
Or maybe this is a typescript rule i was not aware of in term of types

**Review by Greenlamp2 — APPROVED** [review]

**Pull request merged**